pages/facts.html

```diff
@@ -4,9 +4,47 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="../stylesheets/styles.css">
+  <link rel="stylesheet" href="../stylesheets/facts.css">
   <title>Document</title>
 </head>
 <body>
+    <header>
+    <h1>Fun Facts</h1>
+    </header>
+    <section class="container">
+        <div>
+            <h3>Indoor plants improve air quality</h3>
+        </div>
+        <div>
+            <h3>Plants can filter sound  and aborb</h3>
+        </div>
+        <div>
+          <h3> Indoor plants can sense light</h3>
+        </div>
+        <div>
+            <h3>Plants can communicate through root system and emit chemical signals</h3>
+        </div>
+        <div>
+            <h3>Plants can touch sense</h3>
+        </div>
+        <div>
+            <h3>Indoor plants remove toxins from air</h3> 
+        </div>
+        <div>
+            <h3>Indoor plants helps boost mood and reduce stress</h3> 
+        </div>
```

Comment:
I recommend replacing these nested divs with semantic tags like `article`. Where else across the files do you see `div`s that could be replaced with a semantic tag?